subsys/bluetooth/controller/include/ll.h

```diff
@@ -206,3 +206,38 @@ void ll_rx_mem_release(void **node_rx);
 void ll_timeslice_ticker_id_get(uint8_t * const instance_index, uint8_t * const user_id);
 void ll_radio_state_abort(void);
 uint32_t ll_radio_state_is_idle(void);
+
+/* Direction Finding */
+
+/* @brief Function sets CTE transmission parameters for a connection.
+ *
+ * @param[in]handle			Connection handle.
+ * @param[in]cte_types			Bitfield holding information about
+ *					allowed CTE types.
+ * @param[in]switching_patterns_len	Numbe of antenna IDs in switching
```

Comment:
I think this should be `switch_pattern_len` for consistency.
Also missing 'r' in Number
```suggestion
 * @param[in]switch pattern len     Number of antenna IDs in switch pattern.
```

Reply:
Corrected to: "Number of antenna ids in switch pattern".
Name changed to `switch_pattern_len`.